test deps are too conrete or irrelevant (doc build deps)

--- httpretty-0.8.14/development.txt	2016-09-06 18:59:59.745675091 +0300
+++ httpretty-0.8.14/development.txt	2016-09-06 19:01:36.844074805 +0300
@@ -2,16 +2,16 @@
-flake8==2.5.1
-httplib2==0.9.2
-ipdb==0.8.1
-misaka==2.0.0
-mock==1.3.0
-nose==1.3.7
-nose-randomly==1.2.0
-rednose==0.4.3
-requests==2.8.1
-steadymark==0.7.2
-sure==1.2.24
-urllib3==1.12
-tornado==4.3
-coverage==4.0.3
-Sphinx==1.3.3
-sphinx-rtd-theme==0.1.9
+#flake8==2.5.1
+#httplib2==0.9.2
+#ipdb==0.8.1
+#misaka==2.0.0
+#mock==1.3.0
+#nose==1.3.7
+#nose-randomly==1.2.0
+#rednose==0.4.3
+#requests==2.8.1
+#steadymark==0.7.2
+sure>=1.2.24
+#urllib3==1.12
+#tornado==4.3
+#coverage==4.0.3
+#Sphinx==1.3.3
+#sphinx-rtd-theme==0.1.9
